pkg/sdk/v1beta1/kafkacluster_types.go

```diff
@@ -601,10 +601,10 @@ func (cConfig *CruiseControlConfig) GetCCLog4jConfig() string {
 		return cConfig.Log4jConfig
 	}
 	return `log4j.rootLogger = INFO, FILE
-    log4j.appender.FILE=org.apache.log4j.FileAppender
-    log4j.appender.FILE.File=/dev/stdout
-    log4j.appender.FILE.layout=org.apache.log4j.PatternLayout
-    log4j.appender.FILE.layout.conversionPattern=%-6r [%15.15t] %-5p %30.30c %x - %m%n`
+log4j.appender.FILE=org.apache.log4j.FileAppender
```

Comment:
Even though this is not what this PR is for, this log4j.properties seems odd to me.
Combining a file appender with `/dev/stdout` is suboptimal as log4j has a `ConsoleAppender` which is built for this particular purpose (supposedly containing more optimisation regarding pushing data to stdout).

Reply:
This is fair - it's probably better to move this change is a separate pr

Reply:
Added https://github.com/banzaicloud/kafka-operator/issues/496 to track this @adamantal

Reply:
Thanks!